### PR TITLE
translation oneflow.utils.* by lzy 0107

### PR DESCRIPTION
**Pull request by S882li**
开始翻译Oneflow.utils.*

[oneflow.utils.data.BatchSampler](https://github.com/Oneflow-Inc/oneflow/blob/8c07ee58f6bd425573f21d7060f3b925d4a67b15/python/oneflow/utils/data/sampler.py#L192)
image

[oneflow.utils.data.ConcatDataset](https://github.com/Oneflow-Inc/oneflow/blob/79ad53f39246ead1263b40f7e14b212a29492960/python/oneflow/utils/data/dataset.py#L212)
image

[oneflow.utils.data.DataLoader](https://github.com/Oneflow-Inc/oneflow/blob/8c07ee58f6bd425573f21d7060f3b925d4a67b15/python/oneflow/utils/data/dataloader.py#L127)
image
image

1.12更新：
- 修复了0.6.0上线后的一系列问题，包括oneflow.nn.functional.conv1d, oneflow.scatter_nd和oneflow.randperm的代码块问题
- 继续翻译oneflow.utils

[oneflow.utils.data.Dataset](https://github.com/Oneflow-Inc/oneflow/blob/79ad53f39246ead1263b40f7e14b212a29492960/python/oneflow/utils/data/dataset.py#L57)
image

[oneflow.utils.data.IterableDataset](https://github.com/Oneflow-Inc/oneflow/blob/79ad53f39246ead1263b40f7e14b212a29492960/python/oneflow/utils/data/dataset.py#L80)
image
image

[oneflow.utils.data.RandomSampler](https://github.com/Oneflow-Inc/oneflow/blob/8c07ee58f6bd425573f21d7060f3b925d4a67b15/python/oneflow/utils/data/sampler.py#L89)
image

[oneflow.utils.data.Sampler](https://github.com/Oneflow-Inc/oneflow/blob/8c07ee58f6bd425573f21d7060f3b925d4a67b15/python/oneflow/utils/data/sampler.py#L25)
image

[oneflow.utils.data.SequentialSampler](https://github.com/Oneflow-Inc/oneflow/blob/8c07ee58f6bd425573f21d7060f3b925d4a67b15/python/oneflow/utils/data/sampler.py#L71)
image

[oneflow.utils.data.Subset](https://github.com/Oneflow-Inc/oneflow/blob/79ad53f39246ead1263b40f7e14b212a29492960/python/oneflow/utils/data/dataset.py#L295)
image

[oneflow.utils.data.SubsetRandomSampler](https://github.com/Oneflow-Inc/oneflow/blob/8c07ee58f6bd425573f21d7060f3b925d4a67b15/python/oneflow/utils/data/sampler.py#L169)
image

[oneflow.utils.data.TensorDataset](https://github.com/Oneflow-Inc/oneflow/blob/79ad53f39246ead1263b40f7e14b212a29492960/python/oneflow/utils/data/dataset.py#L190)
image

### CODE REVIEW

File: docs/source/cn/utils.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+)
+
+reset_docstr(
+    oneflow.utils.data.TensorDataset,
+    r"""包装张量的数据集。
+
+    每个样本都将通过索引张量的第一个维度来提取。
+
+    Args:
+        - ***tensors** (tensor): 第一维度大小相同的张量
```
[the remark this answers is not in the document]
Comment (S882li) [reply docs/source/cn/utils.py]: 已解决

File: docs/source/cn/utils.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    """
+)
```
[the remark this answers is not in the document]
Comment (S882li) [reply docs/source/cn/utils.py]: 收到

File: docs/source/cn/utils.py [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+    """
+    ,
+)
+
+reset_docstr(
+    oneflow.utils.data.DataLoader,
+
+    r"""
+    数据加载器。结合一个数据集和采样器，并为数据集提供一个可迭代对象。
+    数据加载器同时支持map-style和iterable-style数据集的单/多线程加载，且可以自定义加载顺序和可选的自动处理和内存锁定。
```
[the remark this answers is not in the document]
Comment (S882li) [reply docs/source/cn/utils.py]: 已解决

File: docs/source/cn/utils.py [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.utils.data.BatchSampler,
+    r"""
+    将另一个采样器包装为mini-batch索引。
+
+    参数：
+        - **sampler** (Sampler or Iterable): 原采样器。可以是任何可迭代对象。 
+        - **batch_size** (int): mini-batch的大小
```
[the remark this answers is not in the document]
Comment (S882li) [reply docs/source/cn/utils.py]: 已解决

File: docs/source/cn/utils.py [hunk 5]
```diff
@@ -0,0 +1,216 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.utils.data.BatchSampler,
+    r"""
+    将另一个采样器包装为mini-batch索引。
```
[the remark this answers is not in the document]
Comment (S882li) [reply docs/source/cn/utils.py]: 已解决

**Review by doombeaker — APPROVED** [review]

**Pull request merged**